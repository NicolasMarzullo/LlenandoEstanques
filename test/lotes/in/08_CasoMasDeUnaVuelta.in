5 
200 3 2 
80 6 4 
150 5 3 
70 6 5
20 8
2400 