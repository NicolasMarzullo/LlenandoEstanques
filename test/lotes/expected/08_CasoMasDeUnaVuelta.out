5
1 2.980769231
2 5.980769231
3 4.980769231
4 5.980769231
5 7.980769231